src/components/plot-list/index.tsx

```diff
@@ -47,14 +51,14 @@ export class PlotListBase extends React.PureComponent<PlotListProps, any> {
           bookmark={bookmark}
         />
       );
-    });
-
+    }) : null;
     return (
-      <div>
+      <div style={{position: 'relative'}}>
+        <ClipLoader color={SPINNER_COLOR} loading={isLoading}/>
```

Comment:
Instead of overlaying the spinner with absolute position?  Can't we just show the spinner (with min-height equal to the plot height) when there is no item and show the items otherwise?